extensions/amp-accordion/1.0/accordion.js

```diff
@@ -260,9 +278,14 @@ export function AccordionSection({
 
   useLayoutEffect(() => {
     if (registerSection) {
-      return registerSection(id, defaultExpanded);
+      return registerSection(id, defaultExpanded, onExpand, onCollapse);
     }
-  }, [registerSection, id, defaultExpanded]);
+  }, [registerSection, id, defaultExpanded, onExpand, onCollapse]);
+
+  const expanded = isExpanded ? isExpanded(id, defaultExpanded) : expandedState;
```

Comment:
Why not just have these at the very top of the component?

Reply:
Thanks, moved them towards the top of the component (after the Context part).